packages/ndla-ui/src/NDLAFilm/FilmSlideshow.tsx

```diff
@@ -148,62 +99,59 @@ const FilmSlideshow = ({ slideshow }: Props) => {
   const [currentSlide, setCurrentSlide] = useState(slideshow[0]);
 
   return (
-    <CarouselAutosize breakpoints={slideshowBreakpoints} itemsLength={slideshow.length}>
-      {(autoSizedProps) => (
-        <section>
-          <StyledSafeLink to={currentSlide.path} tabIndex={-1} aria-hidden>
-            <StyledImg src={currentSlide.metaImage?.url ?? ''} alt={currentSlide.metaImage?.alt ?? ''} />
-            <SlideInfoWrapper>
-              <InfoWrapper>
-                <h3>{currentSlide.title}</h3>
-                <span id="currentMovieDescription">{currentSlide.metaDescription}</span>
-              </InfoWrapper>
-            </SlideInfoWrapper>
-          </StyledSafeLink>
-          <CarouselContainer>
-            <Carousel
-              leftButton={
-                <SlideshowButton aria-label={''}>
-                  <ChevronLeft />
-                </SlideshowButton>
-              }
-              rightButton={
-                <SlideshowButton aria-label={''}>
-                  <ChevronRight />
-                </SlideshowButton>
-              }
-              items={slideshow.map((movie) => (
-                <FilmCard
-                  key={movie.id}
-                  current={movie.id === currentSlide.id}
-                  movie={movie}
-                  columnWidth={autoSizedProps.columnWidth}
-                  setCurrentSlide={() => setCurrentSlide(movie)}
-                />
-              ))}
-              {...autoSizedProps}
+    <section>
+      <StyledSafeLink to={currentSlide.path} tabIndex={-1} aria-hidden>
+        <StyledImg
+          src={currentSlide.metaImage?.url ?? ''}
+          sizes="(min-width: 1140px) 1140px, (min-width: 720px) 100vw, 100vw"
+          alt=""
+        />
+        <SlideInfoWrapper>
+          <InfoWrapper>
+            <h3>{currentSlide.title}</h3>
+            <span id="currentMovieDescription">{currentSlide.metaDescription}</span>
+          </InfoWrapper>
+        </SlideInfoWrapper>
+      </StyledSafeLink>
+      <CarouselContainer>
+        <Carousel
+          leftButton={
+            <SlideshowButton aria-label={''}>
```

Comment:
Kan denne settes slik?
```suggestion
            <SlideshowButton aria-label="">
```

Reply:
Kan det! Dette er bare flyttet kode, så jeg har ikke gjort noen endringer på den. Er faktisk litt usikker på hva vi skal gjøre med disse knappene. Burde de egentlig være tilgjengelig for brukere med skjermleser i det hele tatt? Mistenker at de bare kommer til å gi en bruker en dårligere opplevelse enn tabbing.